pkgs/os-specific/linux/kernel-headers/no-dynamic-cc-version-check.patch

```diff
@@ -1,32 +1,12 @@
-diff --git a/Makefile b/Makefile
```

Comment:
This patch may not be necessary any more, but I'm not sure.

Reply:
I have checked that deleting this patch has no effect on the result of `pkgsCross.gnu64.linuxHeaders`, so I've deleted it.